flake8_strict.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E266,W503,E722,E731,E741
+per-file-ignores = */api.py:F401
```

Comment:
Nitpick: missing newline

Reply:
```suggestion
per-file-ignores = */api.py:F401

```

Reply:
(I just tried to fix this using a suggestion, but it's not clear whether that's possible.)